### DISCUSSION TITLE
[blocks]: Gantt charts!

### DISCUSSION THREAD

**Discussion by AlaaSubora** [shadcn-ui/ui#3828] [Blocks Request]
### Description

I would very much appreciate a Gantt chart block.

**Comment by ethosperformance** [comment]
me too!

**Comment by Gallevy** [comment]
Yup, me too

**Comment by EarliestFall988** [comment]
I'd love to collaborate - has anyone already started working on this?

**Comment by LuqmanAlattas** [comment]
you mean similar to this [this](https://recharts-gantt-chart.rdsx.dev/)?

**Reply by EarliestFall988** [reply]
Let's goooo!!

TBH I haven't gotten to this yet lol I'm buried with a few other things. But it looks like it's headed in the right direction.

I'll definitely have to check it out and see how I can contribute!

Thanks for sharing!!

**Reply by 0-don** [reply]
this looks good, but missing drag & drop support

**Comment by Veronika-Savinova** [comment]
me too!

**Comment by 99mill** [comment]
me three!

**Comment by lequanphat** [comment]
me four!

**Comment by agent33894** [comment]
I would love a Gantt chart component! If that was included here I could die happy. 😂
